src/js/views/general.js

```diff
@@ -11,40 +11,19 @@
     template: BDT.templates['general'],
 
     events: {
-      'change [name="inject"]': 'toggleInjection',
       'change [name="tooltips"]': 'toggleTooltips',
       'change [name="timeout"]': 'updateTimeout'
     },
 
     render: function() {
       this.$el.empty().append(this.template());
-      BDT.page.eval('isInjectionEnabled', [], function(enabled) {
-        $('[name="inject"]').prop('checked', enabled);
-        BDT.page.eval('getTimeout', [], function(ms){
-          if (ms !== undefined) {
-            $('[name="timeout"]').val(ms/1000);
-          }
-        })
-      });
 
       BDT.page.eval('areTooltipsEnabled', [], function(enabled) {
         $('[name="tooltips"]').prop('checked', enabled);
       });
       return this;
     },
 
-    toggleInjection: function(evt) {
-      var on = (this.$(evt.currentTarget).is(':checked'));
-      
-      BDT.page.eval(
-        on ? 'enableInjection' : 'disableInjection',
-        [],
-        function() {
-          chrome.devtools.inspectedWindow.reload();
-        }
-      );
-    },
-
     toggleTooltips: function(evt) {
```

Comment:
@yvan33, I also think that tooltips aren't a good deal here, since they flood the UI with too much space.